src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	String getPrintableString();
```

Comment:
think of an appropriate class to implement this method and get the printable string in te format specified in the exercise. no need to resubmit the PR though.